index.js

```diff
@@ -53,6 +53,10 @@ export default class CheapRuler {
         return new CheapRuler(lat, units);
     }
 
+    static units() {
+        return factors;
+    }
+
```

Comment:
That's not equivalent — now you access units with `CheapRuler.units()` rather than `CheapRuler.units`, breaking compatibility. Perhaps we should add a unit test that checks for these values.

Reply:
You're right, I forgot to add `static get units()` (which makes it a read only variable).

Strange that the Typescript definition is written like so:

```ts
public static units: Factors;
```